modules/auxiliary/gather/apache_karaf_command_execution.rb

```diff
@@ -71,16 +71,17 @@ def cmd
   end
 
   def do_login(user, pass, ip)
+    factory = Rex::Socket::SSHFactory.new(framework,self, datastore['Proxies'])
```

Comment:
Any chance we create an Msf::Exploit::Remote::SSH namespace for this and other convenience methods?